test-runner.sh

```diff
@@ -4,6 +4,7 @@ set -o pipefail
 
 get_changes ()
 {
+	git remote remove current && \
```

Comment:
soooo it turns out... adding this line causes the diff check that we do (that would run your code) to fail


https://travis-ci.org/tastejs/todomvc/builds/131036325#L448


The fallback behavior in this case is to run backbone ... thus it is passing :)